client-v2/src/components/FrontsEdit/Edit.tsx

```diff
@@ -117,7 +146,11 @@ const mapStateToProps = (state: State, props: Props) => ({
     state,
     props.match.params.priority || ''
   ),
-  isCurrentFrontsMenuOpen: selectIsCurrentFrontsMenuOpen(state)
+  isCurrentFrontsMenuOpen: selectIsCurrentFrontsMenuOpen(state),
+  feedFontSize:
+    selectOpenFrontsCount(state, props.match.params.priority) > 1
```

Comment:
Is it worth keeping the idea of 'size' abstract and having individual components interpret it? a feedSize of 's', 'm', 'l' could be used by other components if we wanted to make other changes based on front count etc.

Reply:
I did wonder that but it's such a small deviation that I thought `s` and `m` didn't cover it but happy to revisit it, if you think that makes sense!